src/Autosuggest.js

```diff
@@ -18,6 +20,12 @@ export default class Autosuggest extends Component { // eslint-disable-line no-s
     id: PropTypes.string                    // Used in aria-* attributes. If multiple Autosuggest's are rendered on a page, they must have unique ids.
   }
 
+  componentWillReceiveProps(nextProps) {
+    if (typeof nextProps.value === 'string') {
+      this.onUpdate(nextProps.value);
```

Comment:
What do you think about changing `this.onUpdate()` to `this.updateValue()`? I feel it's a bit more self descriptive this way. (I used `onSomething()` functions in handlers, e.g. `onBlur={this.onInputBlur}`, but here there is no handler.)